kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -3,6 +3,8 @@ import { createTranslator } from 'kolibri.utils.i18n';
 export const coreStrings = createTranslator('CommonCoreStrings', {
   // actions
   cancelAction: 'Cancel',
+  changesSaved: 'Changes saved',
```

Comment:
Is this rebased on the latest of `develop`? There should be two messages `changesSavedNotification` and `changesSavedNotification` with these messages?

Reply:
Nope, it was branched out last Thursday. Just done it and replaced these strings